### PR TITLE
Fix appendToJson function when value is a const char*

### PR DESCRIPTION
**Pull request by nesc1**

### CODE REVIEW

File: include/tgbot/TgTypeParser.h [hunk 1]
```diff
@@ -494,6 +494,13 @@ class TGBOT_API TgTypeParser {
         json += ',';
     }
 
+    inline void appendToJson(std::string& json, const std::string& varName, const char* value) const {
+        if(value!=nullptr){
+            std::string strvalue(value);
+            appendToJson(json,varName,strvalue);
+        }
```
Comment (JellyBrick) [inline include/tgbot/TgTypeParser.h:501]:
```suggestion
        if (value != nullptr){
            std::string strValue(value);
            appendToJson(json, varName, strValue);
        }
```

**Review by JellyBrick — CHANGES_REQUESTED** [review]

**Review by JellyBrick — APPROVED** [review]

**Comment by reo7sp** [comment]
CI checks failed. https://travis-ci.org/github/reo7sp/tgbot-cpp/builds/739867367

GCC log:
```
In file included from /usr/src/tgbot-cpp/include/tgbot/Api.h:4:0,
                 from /usr/src/tgbot-cpp/src/Api.cpp:1:
/usr/src/tgbot-cpp/include/tgbot/TgTypeParser.h:504:17: error: 'void TgBot::TgTypeParser::appendToJson(std::__cxx11::string&, const string&, const char*) const' cannot be overloaded
     inline void appendToJson(std::string& json, const std::string& varName, const char *value) const { appendToJson(json, varName, std::string(value)); }
                 ^~~~~~~~~~~~
/usr/src/tgbot-cpp/include/tgbot/TgTypeParser.h:497:17: error: with 'void TgBot::TgTypeParser::appendToJson(std::__cxx11::string&, const string&, const char*) const'
     inline void appendToJson(std::string& json, const std::string& varName, const char* value) const {
                 ^~~~~~~~~~~~
In file included from /usr/src/tgbot-cpp/include/tgbot/Api.h:4:0,
                 from /usr/src/tgbot-cpp/include/tgbot/Bot.h:4,
                 from /usr/src/tgbot-cpp/src/Bot.cpp:2:
/usr/src/tgbot-cpp/include/tgbot/TgTypeParser.h:504:17: error: 'void TgBot::TgTypeParser::appendToJson(std::__cxx11::string&, const string&, const char*) const' cannot be overloaded
     inline void appendToJson(std::string& json, const std::string& varName, const char *value) const { appendToJson(json, varName, std::string(value)); }
                 ^~~~~~~~~~~~
/usr/src/tgbot-cpp/include/tgbot/TgTypeParser.h:497:17: error: with 'void TgBot::TgTypeParser::appendToJson(std::__cxx11::string&, const string&, const char*) const'
     inline void appendToJson(std::string& json, const std::string& varName, const char* value) const {
                 ^~~~~~~~~~~~
```

File: include/tgbot/TgTypeParser.h [hunk 2]
```diff
@@ -494,6 +494,13 @@ class TGBOT_API TgTypeParser {
...
+    }
+
     inline void appendToJson(std::string& json, const std::string& varName, const char *value) const { appendToJson(json, varName, std::string(value)); }
```
Comment (JellyBrick) [inline include/tgbot/TgTypeParser.h:504]:
```suggestion

```

File: include/tgbot/TgTypeParser.h [hunk 3]
```diff
@@ -494,6 +494,13 @@ class TGBOT_API TgTypeParser {
...
     }
 
+    inline void appendToJson(std::string& json, const std::string& varName, const char* value) const {
+        if (value != nullptr){
+            std::string strValue(value);
+            appendToJson(json, varName, strValue);
+        }
+    }
+
     inline void appendToJson(std::string& json, const std::string& varName, const char *value) const { appendToJson(json, varName, std::string(value)); }
```
Comment (JellyBrick) [inline include/tgbot/TgTypeParser.h:504]:
```suggestion
```

**Review by JellyBrick — CHANGES_REQUESTED** [review]

**Comment by reo7sp** [comment]
OK

**Pull request merged**